java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -1291,4 +1291,19 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_title(JNIEnv *env, jobj
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_nansToNulls(JNIEnv *env, jobject j_object, jlong handle) {
+
+  JNI_NULL_CHECK(env, handle, "native view handle is null", 0)
+
+  try {
```

Comment:
All JNI entry-point methods must call `auto_set_device` to set the CUDA device, if necessary.
```suggestion
  try {
    cudf::jni::auto_set_device(env);
```